tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot

```diff
@@ -448,6 +456,7 @@ Provoke Image Build Failure
     Wait Until Build Status Is    namespace=${namespace}    build_name=${failed_build_name}
     ...    expected_status=Running
     Wait Until Container Exist  namespace=${namespace}  pod_name=${pod_name}  container_to_check=${container_to_kill}
+    Sleep    60s    reason=Waiting extra time to make sure the container has started
```

Comment:
isn't possible to check the container is running in a more efficient way? like using Oc library

Reply:
I've been looking at this but it wasn't trivial. Also, this test takes about 30mins, so it won't be a big difference. I think I prefer not to change it in this PR

Reply:
that's fine